CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 - [FEATURE] Add a binary implementation of the HDFS backend (#537)
+- [FEATURE] Add support to batch processing within a transaction (#554)
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/b13eedc0daf7fb440f3d3decb43b596a54cb9f1f